lib/export-scala-base.js

```diff
@@ -91,9 +92,13 @@ const perBusInterfaceParams = comp => e => get(
   () => e => `case class P${e.name}Params() // name: ${e.busType.name}, mode: ${e.interfaceMode}`
 )(comp)(e);
 
+const outConnect = (lhs, rhs) => lhs + ' := blackbox.io.' + rhs;
+const inConnect = (lhs, rhs) => 'blackbox.io.' + rhs + ' := ' + lhs;
+const inoutConnect =  (lhs, rhs) => 'blackbox.io.' + rhs + ' <> ' + lhs;
```

Comment:
Nit: This might be easier to read using interpolated strings, since it removes the need to add in all the `+` operators and opening and closing quotes.

```js
const outConnect = (lhs, rhs) => `${lhs} := blackbox.io.${rhs}`;
```

These are called [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) in JavaScript.